src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -1,585 +1,518 @@
 import {
-    ComboBox,
     DefaultButton,
     Dialog,
     DialogFooter,
```

Comment:
I'm seeing a ton of failed requests when I try to run the environment picker.
![image](https://user-images.githubusercontent.com/57726991/184447224-5f8c1579-830f-4b0f-a2f1-a91fc6425071.png)

Reply:
i think thats sort of a function of how the ARM APIs work (I use the word "work" very loosely, because the APIs are a major PITA)